src/export/export.go

```diff
@@ -53,6 +53,22 @@ func ToDir(state *core.BuildState, dir string, targets []core.BuildLabel) {
 			log.Fatalf("Failed to copy preloaded build def %s: %s", preload, err)
 		}
 	}
+
+	exportPlzConf(dir)
+}
+
+func exportPlzConf(dir string) {
+	profiles, err := filepath.Glob(".plzconfig*")
+	log.Fatalf("failed to glob .plzconfig files: %v", err)
+	for _, file := range append(profiles, ".plzconfig") {
+		path := filepath.Join(dir, file)
+		if err := os.RemoveAll(path); err != nil {
+			log.Fatalf("failed to copy .plzconfig file %v: %v", file, err)
```

Comment:
```suggestion
			log.Fatalf("failed to copy .plzconfig file %s: %v", file, err)
```

Nit, and I don't think this matters at all (certainly doesn't change the behaviour), but using `%v` for a string feels wrong...